share/share.go

```diff
@@ -21,6 +22,9 @@ const (
 // rsmt2d.ExtendedDataSquare.
 var MaxSquareSize = appconsts.SquareSizeUpperBound(appconsts.LatestVersion)
 
+// TailPadding is a constant tail padding share exported for reuse
+var TailPadding Share
```

Comment:
What about make this as a func? `GetTailPadding() Share` by example.

Reply:
What would that give us?

Reply:
Global exported variable is worse than an exported func. You can change behaviour with less pain in the future. +variable gives you ability to change it from another package, `constant` in the comment is no-op.

Reply:
Ah I see you point. I thought you also meant copying the share for every caller.

We do functions in a similar case with empty eds, for the reasons you mentioned. I am gonna do this here as well. Thanks for catching it